README.md

```diff
@@ -65,6 +67,4 @@ If you use Veracruz or otherwise wish to discuss the project, please cite the pr
 
 ## License
 
-This project's codebase is licensed under the [MIT license](LICENSE_MIT.markdown).
-
-We use the image [Veracruz Puerto - Vista desde el Hotel Emporio](examples/data-generators/image-processing-generator/veracruz.jpg) by Eduardo Pavon as a test dataset for some of our examples.  This image is licensed under the [Creative Commons Attribution Share Alike 2.0 Generic (CC-BY-SA 2.0) license](LICENSE_CC-BY-SA-2.0.markdown).
+This project's codebase is licensed under the [MIT license](LICENSE.md).
```

Comment:
Perhaps it would be good to write: Except where stated otherwise, this project's codebase is ...

Reply:
Since the image is not part of the *codebase*, I thought the statement was still true.

Reply:
You're probably right.

Reply:
It does not hurt to add it.